src/agents/common.jl

```diff
@@ -297,7 +312,7 @@ function solve_equilibrium_problem!(
 
 
     TimerOutputs.@timeit HEM_TIMER "solve_equilibrium_problem!" begin
-        for w in 1:5 #(length(model_data.index_y_fix) - window_length + 1)  # loop over windows
+        for w in 1:10 #(length(model_data.index_y_fix) - window_length + 1)  # loop over windows
```

Comment:
This was just for testing (before I changed some parameters there were some instances where I was not able to run until the end of the simulation years).

Reply:
Can this hard coding be removed before merging, or should we open an issue?

I'm also not sure what's going on here, such that it would be hard for me to describe it in an issue. What do you mean by, "there were some instances where I was not able to run until the end of the simulation years"?)

Reply:
Elaine to revert to old code (not hard-coded).